testing-overhead/src/test/java/io/opentelemetry/util/JFRUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.util;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.function.BiFunction;
+import io.opentelemetry.results.AppPerfResults.MinMax;
+import jdk.jfr.consumer.RecordedEvent;
+import jdk.jfr.consumer.RecordingFile;
+
+public class JFRUtils {
+
+  public static long totalLongEvents(Path jfrFile, String eventName,
+      String longValueKey) throws IOException {
+    return reduce(jfrFile, eventName, longValueKey, 0L, Long::sum);
+  }
+
+  public static float findMaxFloat(Path jfrFile, String eventName, String valueKey) throws IOException {
+    return reduce(jfrFile, eventName, valueKey, 0.0f, (BiFunction<Float, Float, Float>) Math::max);
+  }
+
+  public static MinMax findMinMax(Path jfrFile, String eventName, String valueKey) throws IOException {
+    return reduce(jfrFile, eventName, valueKey, new MinMax(), (MinMax acc, Long value) -> {
+      if (value > acc.max) {
+          acc.max = value;
+        }
+        if (value < acc.min) {
+          acc.min = value;
+        }
+        return acc;
```

Comment:
It'd be more functional-style to return a new `MinMax` each time (since you're already using functional programming concepts like `reduce()` function 😛 ); that'd also solve one minor problem with `AppPerfResults`: it appears to be an immutable value class but in fact isn't, because `MinMax` is mutable.

Reply:
ooh, I like this. Lemme do that. 👍🏻